app/lib/theme/widgets/DecimalWidget.tsx

```diff
@@ -10,17 +10,38 @@ export const DecimalWidget: React.FC<WidgetProps> = ({
   label,
```

Comment:
I'm not confident `DecimalWidget` is the best name for this widget--`NumberWidget`?